app/controllers/report_results_controller.rb

```diff
@@ -103,8 +103,14 @@ def create
     if @report.has_pit_options?
       pit_date = @result.options['pit_date'].to_date
       chronic_date = @result.options['chronic_date'].to_date
+      project_ids = @result.options.try(:[], 'project_ids')&.reject(&:blank?)&.map(&:to_i) || []
+      project_group_ids = @result.options.try(:[], 'project_group_ids')&.reject(&:blank?)&.map(&:to_i) || []
+      project_ids += GrdaWarehouse::ProjectGroup.
+        joins(:projects).
+        where(id: project_group_ids).pluck(p_t[:id])
       options[:pit_date] = pit_date
       options[:chronic_date] = chronic_date
+      options[:project_ids] = project_ids
```

Comment:
When you display this later, I _think_ it will include all of the project names from the project groups as well, not sure that's what we want, but probably won't hurt...

Reply:
Doesn't seem to -- it displays the options field on the result object, not the ones passed to the job.